pkg/sidecar/config.go

```diff
@@ -30,6 +30,11 @@ type Config struct {
 	SvidBundleFileNameDeprecated       string `hcl:"svidBundleFileName"`
 	RenewSignal                        string `hcl:"renew_signal"`
 	RenewSignalDeprecated              string `hcl:"renewSignal"`
+
+	// JWT configuration
+	JwtAudience  string `hcl:"audience"`
+	JSONFilename string `hcl:"json_filename"`
```

Comment:
Can we change this to camel case?

Reply:
I think it's not consistent with this [commit](https://github.com/spiffe/spiffe-helper/commit/a3f3ed3a7bda789eb40e73e12bb4bd436b336c4b) where configs are changed to snake case and the user is alerted that camel case will be deprecated.

Reply:
Got it,  you are correct.